WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java

```diff
@@ -59,8 +63,9 @@ public void onClick(View v) {
                             SignInFragment.ENTERED_URL_KEY));
                     HelpshiftHelper.getInstance().addMetaData(MetadataKey.USER_ENTERED_USERNAME, extras.getString(
                             SignInFragment.ENTERED_USERNAME_KEY));
+                    origin = (Tag) extras.get(HelpshiftHelper.ORIGIN_KEY);
```

Comment:
a null `origin` is considered an "unknown origin" (and it should not happen). https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2330-track-helpshift-origin/WordPress/src/main/java/org/wordpress/android/util/HelpshiftHelper.java#L95-L97a